pkg/vulnerability/vulnerability.go

```diff
@@ -25,6 +25,19 @@ const (
 	DefaultIgnoreFile = ".trivyignore"
 )
 
+var (
+	primaryURLPrefixes = map[string][]string{
+		vulnerability.Debian:           {"http://www.debian.org", "https://www.debian.org"},
+		vulnerability.Ubuntu:           {"http://www.ubuntu.com", "https://usn.ubuntu.com"},
+		vulnerability.RedHat:           {"https://access.redhat.com"},
+		vulnerability.OpenSuseCVRF:     {"http://lists.opensuse.org", "https://lists.opensuse.org"},
+		vulnerability.SuseCVRF:         {"http://lists.opensuse.org", "https://lists.opensuse.org"},
+		vulnerability.OracleOVAL:       {"http://linux.oracle.com/errata", "https://linux.oracle.com/errata"},
+		vulnerability.NodejsSecurityWg: {"https://www.npmjs.com", "https://hackerone.com"},
+		vulnerability.RubySec:          {"https://groups.google.com"},
+	}
+)
```

Comment:
Now I understand what you had in mind. This is much better and quite impressive to use for fallback. Nice!

Reply:
Thank you for understanding! Yes, I thought it was pretty easy to support other vulnerability IDs as well as AVD and didn't find any reason not to do that. Sometimes it's hard to explain an idea with words and I'm sorry I couldn't tell it well😁 The source code says more than words.